src/Components/ModuleCard.jsx

```diff
@@ -0,0 +1,28 @@
+import primAjutorOutdoor from'../assets/primAjutorOutdoor.png'
+
+function ModuleCard (titluModul, descriereModul) {
```

Comment:
This is not working as the props should be destructured from the props object.
Try the following way of writing the props

```js
function ModuleCard({title, description}){
//...
}

```

Please test the component in of the pages and chef if props are passed correctly.

Also the name of the props should be in english.